TestON/drivers/common/cli/onosclidriver.py

```diff
@@ -1545,6 +1545,27 @@ def getAllIntentsId( self ):
             main.cleanup()
             main.exit()
 
+
+    def FlowStateCount( self, id ):
```

Comment:
Make that flowsAddedCount(). Any plans to extend this function to return all flow Counts in a list?
(added, any, failed, pending_add, pending_remove, removed)